src/pybind/mgr/dashboard/tests/test_osd.py

```diff
@@ -28,7 +25,7 @@ class OsdHelper(object):
 
     @staticmethod
     def _gen_osdmap_tree_node(node_id, node_type, children=None, update_data=None):
-        # type: (int, str, List[int], Dict[str, Any]) -> Dict[str, Any]
+        # type: (int, str, Optional[List[int]], Optional[Dict[str, Any]]) -> Dict[str, Any]
```

Comment:
As this is test code, it's not that critical to keep Py3 consistency, but I'd prefer to see this moved to inline syntax.